.github/workflows/windows.yml

```diff
@@ -65,13 +70,17 @@ jobs:
         with:
           repo_token: ${{ secrets.GITHUB_TOKEN }}
           file: 'install/bin/mavsdk_server_bin.exe'
-          asset_name: 'mavsdk_server_win32.exe'
+          asset_name: mavsdk_server_win_${{ matrix.arch == 'x64' && 'x64' || matrix.arch == 'Win32' && 'x86' || 'arm64' }}.exe
```

Comment:
Wouldn't it be clearer to do something like [this](https://github.com/mavlink/MAVSDK/blob/main/.github/workflows/macos.yml#L24-L41)?

Something like:

```
- platform: x64
  arch: x64
- platform: Win32:
  arch: x86
- platform: ARM64
  arch: arm64 
```

So that it would simply look like `mavsdk_server_win_${{ matrix.arch }}` and the current "arch" could be called `matrix.platform` instead? My understanding is that "Win32" and "ARM64" are passed to "cmake -A", and "-A" sets the "platform name" as per the cmake documentation.

Reply:
Yep, I like that better.